doc/api/fs.md

```diff
@@ -3324,6 +3324,43 @@ a colon, Node.js will open a file system stream, as described by
 Functions based on `fs.open()` exhibit this behavior as well:
 `fs.writeFile()`, `fs.readFile()`, etc.
 
+### `fs.openAsBlob(path)`
```

Comment:
thinking it would be nice to have an optional type here as well?

`fs.openAsBlob(path, 'text/plain')`